tests/unittests/test_distros/test_netbsd.py

```diff
@@ -0,0 +1,21 @@
+import cloudinit.distros.netbsd
+
+import unittest.mock as mock
+
+
+@mock.patch("cloudinit.distros.netbsd.os")
+def test_init_with_pkgin(m_os):
```

Comment:
We've recently added style docs to cloud-init after our switch to pytest. https://cloudinit.readthedocs.io/en/latest/topics/hacking.html#unit-testing


I think these two tests are good candidates for consolidation into a single test with pytest.mark.parameterize which will pass the expected return_value to the mocked os.path.exists. 

We only have a few examples of moving to pytest.mark.parameterize:
tests/unittests/test_render_cloudcfg.py
cloudinit/distros/tests/test_init.py
cloudinit/net/tests/test_init.py

Also we'd like to see tests in a class declaration

Reply:
Done